README.md

```diff
@@ -175,26 +171,22 @@ foreach($weak_references as $weak_ref_object_handle => $weak_reference) {
 
 ## Development and testing
 
-This extension shipped with Vagrant file which provides basic environment for development and testing purposes. 
-To start it, just type `vagrant up` and then `vagrant ssh` in php-weak directory.
+This extension ships with a Vagrant file which provides a basic environment for development and testing purposes. 
+To start it, just type `vagrant up` and then `vagrant ssh` from within the main directory.
 
 Services available out of the box are:
 
  - Apache2 - on [192.168.33.10:8080](http://192.168.33.102:8080)
  - nginx - on [192.168.33.10:80](http://192.168.33.102:80)
 
-For plumbing memory-related problems there are valgrind, to activate it, execute `export TEST_PHP_ARGS=-m` before running tests.
+For debugging memory-related problems, valgrind is already installed. To activate it, execute `export TEST_PHP_ARGS=-m` before running tests.
 
-To prevent asking test suite to send results to PHP QA team, `NO_INTERACTION=1` env variable is set. If run tests in your
```

Comment:
Here I tried to say that out of the box in .bashrc NO_INTERACTION env variable is already set to 1